client/src/common/constants/index.ts

```diff
@@ -15,3 +15,109 @@ export const searchTypes: SearchType[] = [
 ]
 
 export const SUBSPACE_ACC_PREFIX = 2254
+
+export const MODULES = [
```

Comment:
it's probably ok for now to use constants, but ideally, this should be dynamic because runtime will change and there will be different lists of modules

Reply:
I can create a custom query for this on the backend side that will return current all modules

Reply:
Sounds good. Can we have it for this and `Log types` as well?

Reply:
sure

Reply:
This will be addressed on this issue: https://github.com/subspace/blockexplorer/issues/266